src/rebar_mix_elixir_finder_hook.erl

```diff
@@ -41,7 +41,10 @@ do(State) ->
     end.
 
 process_elixir_lib_paths(State) ->
-    case rebar_mix_builder:sh(?ELIXIR_CMD,[{return_on_error,true},{use_stdout,false}]) of
+    case rebar_mix_builder:sh(?ELIXIR_CMD,
+                        [{return_on_error,true},
+                         {use_stdout,false},
+                         {env, [{"ERL_FLAGS", ""}]}]) of
```

Comment:
I added this out of the need to maintain compatibility with previous code, following the change at https://github.com/Supersonido/rebar_mix/pull/22/files#r626461900.